SEED-platform/seed
# Improve inventory document upload process

#### Any background context you want to provide?
The Inventory Detail Document Upload modal had untranslated titles and error messages. Additionally, it had no help text to assist the user.

#### What's this PR do?
- Adds translated titles to each step
- Adds informative error messages
- Adds help text to indicate which file types are accepted 

#### How should this be manually tested?
Go to inventory detail, upload a .dxf, .pdf, .idf, or .osm file. Ensure help text is present and relevant 
Upload an invalid file type (.txt, .xslx) and ensure the error message is informative

#### What are the relevant tickets?
#3838 

#### Screenshots (if appropriate)
New
![Screenshot 2023-02-10 at 9 36 17 AM](https://user-images.githubusercontent.com/58446472/218146435-4ee0a2bf-dc24-4693-b1aa-2501362d927d.png)
![Screenshot 2023-02-10 at 9 36 52 AM](https://user-images.githubusercontent.com/58446472/218146430-70b44cf8-a74e-4d64-a064-a0d854f1ea61.png)
![Screenshot 2023-02-10 at 9 36 33 AM](https://user-images.githubusercontent.com/58446472/218146432-073a6323-1ee8-46d9-9401-8e892b50fd19.png)


Old
![Screenshot 2023-02-10 at 9 40 34 AM](https://user-images.githubusercontent.com/58446472/218146898-6488ea2c-c6dd-455d-baa6-7c6587fa3af7.png)
